.rultor.yml

```diff
@@ -8,7 +8,9 @@ docker:
 merge:
   fast-forward: only
   script:
-    - python3 .misc/adjust_version_number.py coalib/VERSION -b `date --utc +%Y%m%d%H%M%S`
+    - >
+      python3 .misc/adjust_version_number.py
+      coalib/VERSION -b `date --utc +%Y%m%d%H%M%S`
     - bash .misc/check_maintainership.sh
```

Comment:
Remove these too as all these do not belong to the PR & Issue

your commits should only be present in YML file

Reply:
@yashovardhanagrawal These should be here, please read the issue description. The issue needs to fix any errors raised by coala thats why earlier I pointed out to amend the commit message.

Reply:
oh sorry, my fault
I should have realised that

anyway he has to issue a new PR now